extensions/amp-twitter/1.0/component.js

```diff
@@ -48,6 +50,20 @@ function BentoTwitterWithRef(
         const height = data['height'];
         if (requestResize) {
           requestResize(height);
+
+          if (iframeRef.current.node.getRootNode().host) {
+            // Remove the position style from embed after the resize is complete.
+            resetStyles(iframeRef.current.node.getRootNode().host, [
```

Comment:
Can we use a classname with JSS instead of using `setStyles`/`resetStyles`?

Reply:
Thanks, @alanorozco I have replaced the `setStyles/resetStyles` with JSS classes